config/environments/production.rb

```diff
@@ -33,12 +33,14 @@
   # Enable serving of images, stylesheets, and JavaScripts from an asset server.
   # config.asset_host = "http://assets.example.com"
 
+  config.action_mailer.default_url_options = { :host => "https://hotel-app.fly.dev/" }
+
   # Specifies the header that your server uses for sending files.
   # config.action_dispatch.x_sendfile_header = "X-Sendfile" # for Apache
   # config.action_dispatch.x_sendfile_header = "X-Accel-Redirect" # for NGINX
 
   # Store uploaded files on the local file system (see config/storage.yml for options).
-  config.active_storage.service = :local
+  config.active_storage.service = :production
```

Comment:
Bu değişiklik neden ?

Reply:
Yukarıda production ortamı için yapmış olduğum database eklemesi ile birlikte production ortamında bu pathi belirtmeden ekleme işlemi yapmıyordu. Bu nedenle böyle bir değişikliğe ihtiyaç duydum. Tabi  aşağıda belirtmiş olduğun gibi bir değişiklik yapsaydım buraya dokunmama gerek kalmazdı.